spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -133,7 +133,7 @@
         city: 'ANYTOWN',
         state: 'MD',
         dob: '1986-07-01',
-        sex: nil,
+        sex: 'male',
```

Comment:
This changed because this test is using the `true_id_response_success_with_liveness.json` fixture. This fixture was recorded recently so it did not include a `Fields_Sex` attribute. It does, however, contain a sex attribute in the `AUTHENTICATION_RESULT` group. The changes in this PR result in `male` being read from the response for the `sex` value.